Automattic/woocommerce-payments
# Add action column with respond button to dispute list view

Fixes #6667 

#### Changes proposed in this Pull Request

Add action column with CTA `Respond` button. 

As part of a larger set of changes with the disputes list view. This PR adds a CTA to make viewing dispute details (and actioning them) more prominent. 

If a dispute does not need a response due to a previous response or due to going past the due date, a `View dispute` link appears instead.
 
![image](https://github.com/Automattic/woocommerce-payments/assets/57298/9d7d3633-2b69-42c5-8fac-c9421f47c06a)
 

#### Testing instructions

In a store with active and historical disputes.
1. Visit Payments > Disputes
2. See active disputes with `Respond` button.
3. Change filter from `Needs Response` to `All Disputes`
4. See historical disputes with `See dispute` link in action column


-------------------


**Post merge**
